CHANGELOG.md

```diff
@@ -10,6 +10,15 @@ internal API changes are not present.
 v0.44.2 (2024-01-29)
 -------------------------
 
+### Other changes
```

Comment:
^^^ above here, you would want to update this to be `v0.44.3`

otherwise it looks good as I checked with https://github.com/grafana/agent/compare/v0.44.2...main